lib/rubyplot.rb

```diff
@@ -144,12 +144,23 @@ module Rubyplot
     :double_line_single_ended,
     :double_line_double_ended
   ].freeze
-  
-  class << self    
+
+  class << self
+
+    attr_accessor :iruby_inline
```

Comment:
Add docs saying what exactly this does and how it can be used. BTW you were going to push some method-specific docs right?

Reply:
I will work on writing the docs soon and will make a PR for that too.